org.gnome.Shotwell.json

```diff
@@ -0,0 +1,175 @@
+{
+    "app-id": "org.gnome.Shotwell",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "branch": "stable",
+    "sdk": "org.gnome.Sdk",
+    "command" : "shotwell-wrapper",
+    "rename-desktop-file" : "shotwell.desktop",
+    "rename-icon" : "shotwell",
+    "rename-appdata-file" : "shotwell.appdata.xml",
+    "finish-args": [
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.local/share/shotwell",
```

Comment:
Remove this. Flatpaks are supposed to be isolated. (It is also broken if you didn't already have Shotwell on the host).

Reply:
Two potential problems:
 * People expect the flatpaks to access their collection (Communicatable and solvable, I had that with the nighlies as well)
 * If you have had a previous Shotwell installation on the host, it will have generated its so-called backing photos inside the library folder. If people use the flatpak and their library folder (usually $XDG_PICTURES_DIR), they will get duplicated images, I'm afraid. And that's quite annoying.